src/libsync/syncfilestatustracker.cpp

```diff
@@ -229,10 +229,10 @@ void SyncFileStatusTracker::slotAboutToPropagate(SyncFileItemVector &items)
         _dirtyPaths.remove(item->destination());
 
         if (hasErrorStatus(*item)) {
-            _syncProblems[item->_file] = SyncFileStatus::StatusError;
+            _syncProblems[item->destination()] = SyncFileStatus::StatusError;
```

Comment:
Hm is that only an issue here or everywhere where _file is used?

Reply:
_file is usually the same as destination(),  unless this is a move operation, then _file is the source.
But yes, in many case we want destination, while in some other case we want the source.  Perhaps having a source() function to make it explicit and avoid the use of _file should be done.